pkg/controller/util_test.go

```diff
@@ -13,7 +13,7 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	v1 "k8s.io/api/core/v1"
 	storagev1 "k8s.io/api/storage/v1"
-	apiextensionsv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	extv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
```

Comment:
It would be better to name this apiextensions or k8ext, not extv1, we are not using both versions so there is no need to append v1. And then update 'might' be easier if new API is compatible. 

On the other hand I see it is a rule used in all imports. And if we do not want to change the rule then I'll accept this as is.

Reply:
So if we use a v1 of anything, we usually name it something like <package short name>v1, like extv1, or corev1, or storagev1. We can name it whatever, I picked extv1, but there is no rule we can't name it something else.

Reply:
ok